src/gretel_trainer/relational/report/report_template.html

```diff
@@ -3,91 +3,148 @@
+          <span>
+            <b>Generated</b> {{presenter.generated_at}}
+            <br/>
+            <b>Generation ID</b> {{ presenter.run_identifier }}
+          </span>
+        </div>
+
+        <div class="score-container mb-1">
+            <span class="label label__{{ presenter.composite_sqs_score_and_grade[1].lower().replace(' ', '-') }} mb-0.5">
+              {{ presenter.composite_sqs_score_and_grade[1] }}
+            </span>
+            <div id="chart" class="score mb-0.5">
+                <figure>{{ presenter.composite_sqs_figure.to_html(config={'displayModeBar': False}, full_html=False, include_plotlyjs=False) }}</figure>
+            </div>
+            <div>
+                <span class="score-container-text">Composite<br />Synthetic Data Quality Score</span>
+            </div>
+        </div>
+
+        <div class="score-container mb-1">
+            <span class="label label__privacy__{{ presenter.composite_ppl_score_and_grade[1].lower().replace(' ', '-') }} mb-0.5">
+              {{ presenter.composite_ppl_score_and_grade[1] }}
+            </span>
+            <div id="chart" class="score mb-0.5">
+                <figure>{{ presenter.composite_ppl_figure.to_html(config={'displayModeBar': False}, full_html=False, include_plotlyjs=False) }}</figure>
+            </div>
+            <div>
+                <span class="score-container-text">Composite<br />Privacy Protection Level</span>
+            </div>
         </div>
 
+        </div>
     </div>
 
-    <section class="scroll">
+    <section class="scroll test-table-relationships">
       <h2>Table Relationships</h2>
 
       <p>
-        The fields of each table in the synthesized database and their relationships are displayed below. Primary keys
-        are <strong>bolded</strong>. Foreign keys are highlighted in the same color as their corresponding primary key.
+        The primary and foreign keys for each table in the synthesized database and their 
+        relationships are displayed below.
       </p>
 
-
       <table>
-          <thead>
-            <tr>
-              {% for table in presenter.relationships.columns %}
-                <th scope="col">{{table}}</th>
-              {% endfor %}
-            </tr>
-          </thead>
-          <tbody>
-            {% for cell_list in presenter.relationships.data %}
-            <tr>
-              {% for cell in cell_list %}
-                <td style="{{cell.style}}">{{cell.value}}</td>
-              {% endfor %}
-            </tr>
+        <tr>
+            <th>Table Name</th>
+            <th>Primary Key</th>
+            <th>Foreign Keys</th>
+        </tr>
+        {% for table_data in presenter.report_table_data %}
+        <tr>
+          <td><b>{{ table_data.table }}</b></td>
+          <td><span class="label">{{ table_data.pk }}</span></td>
+          <td>
+            {% for fk in table_data.fks %}
+            <span class="label">{{ fk.column_name }}</span>&nbsp;
```

Comment:
So with these label classes, js will add/remove label color classes on hover.  It will be great.